modules/cli/pages/index.adoc

```diff
@@ -178,6 +178,56 @@ $ skupper link delete <link-name>
 ----
 where `<link-name>` is the name of the link specified during creation.
 
+// Type: procedure
+[id="link-cost"] 
+== Specifying link cost
+
+When linking sites, you can assign a cost to each link to influence the traffic flow. By default, link cost is set to `1` for a new link.
+
+If you have more than two sites and services distributed across different clusters, you might want a calling service to favor a particular target or link.
+In this case, you can specify a cost of greater than `1` for links that should be avoided.
```

Comment:
Propose something like (from qpid doc):

In this case, you can specify a cost of greater than `1` to influence the routing algorithm as it attempts to use the path with the lowest total cost from caller to target service.

Reply:
Added https://github.com/skupperproject/skupper-docs/pull/123/files#diff-6c02c30b52aba7f97609f6c67a8d5ffc64ca5fc016f81e08d164a83275c8f3d5R186

I think your point is good, and needs to be promoted above this text